BitwardenShared/UI/Auth/AuthRoute.swift

```diff
@@ -84,6 +88,10 @@ public enum AuthRoute: Equatable {
     ///
     case startRegistration
 
+    /// A route to start registration screen when coming from the expired link view.
```

Comment:
⛏️ To keep the same wording use `screen`:
```suggestion
    /// A route to start registration screen when coming from the expired link screen.
```

Reply:
Yes, if you can and it's not too much work given that's outside of this task scope. Btw, I don't mind choosing either `view` or `screen` as long as we always use the same one.